js/pmpro-login.js

```diff
@@ -12,7 +12,13 @@ jQuery(document).ready(function(){
 		
 		var strength;		
 		if ( pass1 != '' ) {
-			strength = wp.passwordStrength.meter( pass1, wp.passwordStrength.userInputBlacklist(), pass1 );
+			// Call the disallowed list method corresponding to appropriate WP version.
+			const disallowedList = ( 'function' == typeof wp.passwordStrength.userInputDisallowedList )
+				? wp.passwordStrength.userInputDisallowedList()
+				: wp.passwordStrength. userInputBlacklist();

```

Comment:
@ideadude A space snuck in here :(

Reply:
Fixed https://github.com/strangerstudios/paid-memberships-pro/commit/a54ca035d8d1a644ab0863087ecb88fc6b11e700